src/main/java/calculator/domain/Calculator.java

```diff
@@ -0,0 +1,60 @@
+package calculator.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    private static final String REGEX = "//(.)\n(.*)";
+    private static final String DEFAULT_DELIMITERS = ",|:";
+    private static final String ILLEGAL_ARGUMENT_EXCEPTION_CONTENT = "구분자와 양수만 입력 가능합니다.";
+
+    private final List<Integer> numbers;
+
+    private Calculator(List<Integer> numbers) {
+        this.numbers = numbers;
+    }
+
+    public int sum() {
+        return numbers.stream()
+            .mapToInt(Integer::intValue)
+            .sum();
+    }
+
+    public static Calculator from(String input) {
+        input = input.replace("\\n", "\n");
+        Matcher matcher = Pattern.compile(REGEX, Pattern.DOTALL).matcher(input);
+
+        String delimiters = DEFAULT_DELIMITERS;
+        String inputNumbers = input;
+
+        if (matcher.matches()) {
+            delimiters += "|" + Pattern.quote(matcher.group(1));
+            inputNumbers = matcher.group(2);
+        }
+
+        List<Integer> numbers = parseNumbers(inputNumbers, delimiters);
+        return new Calculator(numbers);
+    }
+
+    private static List<Integer> parseNumbers(String inputNumbers, String delimiters) {
+        List<Integer> result = new ArrayList<>();
+        for (String token : inputNumbers.split(delimiters)) {
+            try {
+                if (Objects.equals(token, "")) {
```

Comment:
아래와 같이 사용해보는것은 어떨까요!?
```suggestion
                if (token.isBlank()) {
```

Reply:
자바 기본 함수에 대해 더 공부할 필요가 있겠네요.. 이런 함수가 있었다니 🥲
유익한 정보 감사합니다!